src/hooks/useChainInfo.ts

```diff
@@ -53,6 +54,8 @@ export function useChainInfo(api: ApiPromise) {
     chainInfo.value = createInfo(api, systemChain, systemName, specName);
   });
 
+  const store = useStore();
```

Comment:
do we need store here?

Reply:
Fixed in this commit https://github.com/AstarNetwork/astar-apps/pull/469/commits/ed79f2ae9bf5c316d52f3683f83024e8673eb5ca